### DISCUSSION TITLE
Table Hover with Drag and Drop

### DISCUSSION THREAD

**Discussion by markddrake** [twbs/bootstrap#37980] [Q&A]
I hve a table that represents 6 5 minute slots. I have table set up so that table-hover high-lights the slot the mouse is in. It works fine when I move the mouse betwen slots. However each stot is a dropable target. When I drag something over the slots the table hover does not work. It there a way to use the ondrag events to force the table hover effect ?

**Comment by TommyRiquet** [comment]
There's a way to do that using the dragover eventListener.  
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragover_event

Here's a code that could work with your problem : 
```javascript
const tableCells = document.querySelectorAll('.table-cell');

tableCells.forEach(cell => {
  cell.addEventListener('dragover', event => {
    event.preventDefault();
    cell.dispatchEvent(new Event('mouseenter'));
  });
});

```

This code will prevent the default behaviour and dispatch a mouseenter event  on the cell, effectively triggering the table hover effect.
